bash_completions.go

```diff
@@ -29,14 +30,6 @@ __debug()
     fi
 }
 
-# Homebrew on Macs have version 1.3 of bash-completion which doesn't include
-# _init_completion. This is a very minimal version of that function.
-__my_init_completion()
-{
-    COMPREPLY=()
-    _get_comp_words_by_ref cur prev words cword
-}
```

Comment:
Why did you do this? _init_completion is the right thing to do. You just removed functionality from people who can use it and forced us all to be stuck with what OS X people have...